src/components/common/FilterComponent.jsx

```diff
@@ -54,14 +54,15 @@ class FilterComponent extends React.Component {
             <Form>
               {
                   option.content.map((opt) => {
+                    const newValue = opt.option === null ? 'Unspecified' : opt.option;
```

Comment:
This can just be:
```js
const label = opt.option || 'Unspecified'
```
`null` is a falsy value

Reply:
When I make this change 0 becomes unspecified

![screen shot 2018-11-30 at 13 12 24](https://user-images.githubusercontent.com/31322228/49283020-acc05d00-f4a1-11e8-9a3f-84062d9ae3bc.png)

Reply:
Alright @kimotho-njoki. Then use [isNull](https://lodash.com/docs/4.17.11#isNull) from lodash. This way, when you read the code one week later you will not be thinking of the reason why you didn't use the Logical OR. `isNull` is self-explanatory